test/core/test/jest-expect.test.ts

```diff
@@ -51,6 +51,12 @@ describe('jest-expect', () => {
     expect(new Date(0)).toEqual(new Date(0))
     expect(new Date('inValId')).toEqual(new Date('inValId'))
 
+    expect(new Error('message')).toEqual(new Error('message'))
+    expect(new Error('message')).not.toEqual(new Error('different message'))
```

Comment:
**Comment**: When adding tests, I also added the same kind of cases for `Error` objects. Let me know if you'd rather not have them in this PR.